Tutorials/ASP.NET/WebApi/README.md

```diff
@@ -0,0 +1,119 @@
+## Overview
+
+This tutorial demonstrates how to create an ASP.NET Core Web API service that supports the [WebApiDataStoreClient](http://docs.devexpress.com/XPO/DevExpress.Xpo.DB.WebApiDataStoreClient) provider. For additional information about WebApiDataStoreClient, refer to the following help topic: [Transfer Data via REST API](http://docs.devexpress.com/XPO/402182/connect-to-a-data-store/transfer-data-via-rest-api).
+
+## Prerequisites
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) with the following workloads:
+  * **ASP.NET and web development**
+  * **.NET Core cross-platform development**
+* [.NET Core SDK 5.0 or later](https://www.microsoft.com/net/download/all)
+
+## Step 1: Create an ASP.NET Core Web Api project
+
+1. Open Visual Studio and create a new project.
+
+2. Search for the **ASP.NET Core Web Application** template. Click **Next** to proceed.
```

Comment:
Search... and click next? You do not have to do anything with the **ASP.NET Core Web Application** template? Select it?

Reply:
How about: Select the **ASP.NET Core Web Application template**, and click **Next**.